.github_changelog_generator

```diff
@@ -0,0 +1,21 @@
+# Command to generate: github_changelog_generator --since-tag <last release> --future-release <new release> -t <github token>
+
+# Running the command updates the file with info from rero/rero-ils:staging
+base=CHANGELOG.md
+user=rero
+project=rero-ils
+release-branch=staging
+
+# Limit issues fetched so as to not reach secondary API limit
+max-issues=300
```

Comment:
Difficult to know if this can be an issue.

Reply:
From what I've gathered, it is a problem for generating a complete changelog but not to update an existing one. The first issues to be fetched are the most recent ones. The API limit is hit when fetching commit and issue events at around 1400, a comfortable margin.